packages/dendron-cli/src/commands/base.ts

```diff
@@ -102,6 +106,51 @@ export abstract class CLICommand<
     this.L.info({ msg: `Telemetry is disabled? ${segment.hasOptedOut}` });
   }
 
+  async validateConfig(opts: { wsRoot: string }) {
+    const { wsRoot } = opts;
+    const ws = new WorkspaceService({ wsRoot });
+    const dendronConfig = ws.config;
+    const configVersion = ConfigUtils.getProp(dendronConfig, "version");
+    const clientVersion = CLIUtils.getClientVersion();
+    const validationResp = ConfigUtils.configIsValid({
```

Comment:
its okay to say this is out of scope for this but we should also add a followup to check if the config is **actually** valid for the given version.
a user can keep the verison but manually change the settings. ideally, we shhould use json schema to validate this (lets not use a mess of if statements).
each config version can come with its own json schema and we can use `ajv` to validate

Reply:
yes this is the plan down the line.